com/win32com/src/oleargs.cpp

```diff
@@ -138,7 +138,14 @@ BOOL PyCom_VariantFromPyObject(PyObject *obj, VARIANT *var)
         V_BOOL(var) = VARIANT_FALSE;
     }
     else if (PyLong_Check(obj)) {
+#if PY_VERSION_HEX >= 0x03140000
+        int sign;
+        if (!PyLong_GetSign(obj, sign)) {
+            return NULL
+        }
```

Comment:
My lack of CPP expertise will show here, but according to the doc:
> On failure, return -1 with an exception set.

So iiuc, to properly bubble up that exception, I just have to return `NULL` ? Or is there something else I need to do as well.

Note that the rest of this function always returns `FALSE` when `PyErr_Occurred()`. So maybe I want to return `FALSE` here as well for consistency ?